Enable usage of HttpOnly Cookies to authenticate browsers (#1959)
kuzzleio/kuzzle: doc/2/guides/main-concepts/authentication/index.md

}'
```

> ### Authentication Token in the Browser

Reviewer: Same as above

> When you're making `HTTP request` from a Browser you can instruct Kuzzle

Reviewer 2: ```suggestion
When you're sending HTTP requests from a browser you can instruct Kuzzle
```

to `load` and `store` authentication tokens within an [HTTP Cookie](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies).

> This is possible thanks to the [HTTP-only](../../../api/protocols/http/index.md#http-only-options) option [cookieOnly](../../../api/protocols/http/index.md#cookieonly)

Reviewer 2: ```suggestion
This is possible thanks to the [HTTP-only](/core/2/api/protocols/http#http-only-options) option [cookieOnly](/core/2/api/protocols/http#cookieonly)
```

::: warning

> For this to work you should enable the option `supportCookieAuthentication` in the [Kuzzle Configuration](../../advanced/configuration/index.md)

Reviewer 2: ```suggestion
This only works if the option `supportCookieAuthentication` is enabled in the [Kuzzle Configuration](/core/2/guides/advanced/configuration)
```

}'
```

<SinceBadge version="auto-version"/>
### Authentication Token in the Browser

When you're sending HTTP requests from a browser you can instruct Kuzzle
to `load` and `store` authentication tokens within an [HTTP Cookie](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies).

> If a cookie is stored in the browser

Reviewer 2: dangling sentence?
